latex_table.py

```diff
@@ -17,7 +17,7 @@
     "HippyVM": "HippyVM",
     "PyHyp-mono": "PyHyp$_m$",
     "PyHyp-comp": "PyHyp$_c$",
-    "PyHyp-comp-rev": "PyHyp$_{c'}$",
+    "PyHyp-comp-rev": "PyHyp$_{c2}$",
```

Comment:
If you like. However a few months ago you expressed dislike for the term "reversed".